### DISCUSSION TITLE
Fn+F12 opens a new instance of the calculator app everytime is pressed.

### DISCUSSION THREAD

**Discussion by EmiliManzano** [BartoszCichecki/LenovoLegionToolkit#677] [General]
I would find more logical to open the one that is already running every time you use the Fn + F12 action, to continue with previous operations. 

I don't know if is intended or not :)

**Comment by BartoszCichecki** [comment]
It's up to each program how stuff like this is handled.

**Comment by e-t-l** [comment]
While LLT can't modify this natively, there certainly are apps that could give you your desired outcome by monitoring/restricting process execution. For example, I'm almost certain AutoHotKey can detect whether the Calculator app is running, and if a new Calculator process is launched, to prevent/terminate the new process and then make the existing Calculator process window active.
